### DISCUSSION TITLE
Help with Firestore

### DISCUSSION THREAD

**Discussion by TheCukitoDev** [sdorra/content-collections#342] [Q&A] [answered]
Hello, i am creating a blog with next.js and i were trying to implement mdx components but i couldn't find any solution so i found something like in magicui code (this repo) so i don't know how to implement it. I use firestore that retrieves the post data and then calls a firebase storage bucket to get the mdx file. Can I implement it with content-colections? And how? 
Thanks

**Comment by TheCukitoDev** [comment]
Also, how can i make it work withe sentry??

**Comment by sdorra** [comment, marked as answer]
Sorry, but i don't think that Content Collections is the right tool for your use case. Content Collections is all about collecting local files. There is no way to collect files from a remote location.
